EventFilter/L1TRawToDigi/plugins/implementations_stage2/EMTFBlockSP.cc

```diff
@@ -16,6 +16,13 @@ namespace l1t {
         bool unpack(const Block& block,
                     UnpackerCollections* coll) override;  // Apparently it's always good to use override in C++
         // virtual bool packBlock(const Block& block, UnpackerCollections *coll) override;
+
+      private:
+        bool useHMTBits_{false};
+        bool useNNBits_{false};
+
+        int nominalShower_ = 2;
+        int tightShower_ = 3;
```

Comment:
```suggestion
```
Why `nominalShower_` and `tightShower_` are class members, if they are only used as constants in the `unpack` method?
Please make them `constexpr` and local inside `unpack`